src/common/HPM.c

```diff
@@ -294,13 +298,13 @@ void *hplugins_getFromHPData(enum HPluginDataTypes type, uint32 pluginID, struct
 {
 	int i;
 
+	if (!store)
```

Comment:
it's okay for `store` to be a NULL pointer here (it gets initialized by `HPM->data_store_validate()`).

The check was correct after the call